include/util/command.hpp

```diff
@@ -50,7 +50,7 @@ inline int close(FILE* fp, pid_t pid) {
     ret = waitpid(pid, &stat, WCONTINUED | WUNTRACED);
 
     if (WIFEXITED(stat)) {
-      spdlog::debug("Cmd exited with code {}", WEXITSTATUS(stat));
+      spdlog::trace("Cmd exited with code {}", WEXITSTATUS(stat));
```

Comment:
I changed this (and below) to trace since they made debugging  with `-l debug` very noisy. I'll be happy to revert this and/or open another PR in case we want separation of concerns.